handlers/meta/handlers.go

```diff
@@ -130,7 +131,7 @@ func (h *handler) WriteRequestError(ctx context.Context, w http.ResponseWriter,
 
 // GetChannel returns the channel
 func (h *handler) GetChannel(ctx context.Context, r *http.Request) (courier.Channel, error) {
-	if r.Method == http.MethodGet {
+	if r.Method == http.MethodGet || r.URL.Path == "/c/fba/delete" {
```

Comment:
@norkans7 I'm not getting why we need this..

Reply:
We use the same URL for all channels and this lookup the channel for the message however for the delete request we do not get info about the channel at all to be able to look it up so will ignore and the URN we get will be unique since they page scoped

Reply:
but how can we look up a URN without a channel? Don't we need channel to get org_id ?

Reply:
Right, this will not work without a valid channel, the request I expect we get on that also cannot allow us to lookup the channel

I guess we remove the check for the contacts exists in the DB and rely on the URN that we know if page scoped

Reply:
We can't create a channel event without a channel... and we don't gain much from making it mailroom's problem to determine if a URN exists - we don't have an index on URN without org id. Argh.